### ISSUE TITLE
Increase font size of "Adicione aqui o texto..." and comment lines

### ISSUE THREAD

**Issue by jamesholston** [peer/doc#126]
The current font size is quite small and therefore "Adicione aqui o texto...", "Sem comentários," and "Iniciar um comentário..." remain hard to see.  I would increase the size one or two steps.  Make it easier for those of us who use glasses!
-- To be consistent with the title and body text lines to the right,I would also remove the period from "Sem comentários."  and from "Iniciar um comentário ... esquerda."

<img width="1440" alt="screen shot 2018-06-28 at 5 24 13 pm" src="https://user-images.githubusercontent.com/5572598/42073040-8e4afdac-7b3a-11e8-9e44-15bdf2e8006d.png">

**Comment by mitar** [comment]
> To be consistent with the title and body text lines to the right,I would also remove the period from "Sem comentários." and from "Iniciar um comentário ... esquerda."

No, the right ones are messages, while the left ones are placeholders. It is different, so dots are different.

**Comment by mitar** [comment]
> The current font size is quite small and therefore "Adicione aqui o texto...", "Sem comentários," and "Iniciar um comentário..." remain hard to see. I would increase the size one or two steps. Make it easier for those of us who use glasses!

So I think this is a general question on how large should be the content. Currently the title placeholder is simply heading 1 size, and body placeholder is simply body size. I think you are also saying that once you put in regular text, that text is also too small, or not?

**Comment by mitar** [comment]
I have tested this and compared with Google Doc in the same browser, same machine, the default Google Doc. I opened it and by default it uses Arial of size 11pt for regular text. In reality, the font has font size 14.6667px. We currently do not expose the value in pt to the user, but our default text size for content is 14px. Their menu text is 13px, while our "no comments" text is 14px. Their heading 1 text is 26.6667px, our heading 1 text is 28px.

So we have larger messages and headings, but slightly smaller regular text.

So I would prefer that we keep things like it is because it has size aligned with all other UI elements. Changing one could start breaking the size ratios to the rest.

But I do think we should add an option to increase the font, like Google has it. I will open another issue for that. Which would increase only the font of the content. (If everything is too small, browsers can increase font for you.)
